libkbfs/tlf_journal.go

```diff
@@ -1349,11 +1350,11 @@ func (j *tlfJournal) flushOneMDOp(
 
 	j.log.CDebugf(ctx, "Flushing MD for TLF=%s with id=%s, rev=%s, bid=%s",
 		rmds.MD.TlfID(), mdID, rmds.MD.RevisionNumber(), rmds.MD.BID())
-	pushErr := mdServer.Put(ctx, rmds, extra)
+	pushErr := mdServer.Put(ctx, rmds, extra, nil, keybase1.MDPriorityNormal)
```

Comment:
We might need to do something more here, since Git writes data through the journal and will need to mark it with the git priority and, in some cases, with lock info.  The priority could be a config option that's always applied, but the lock info might need to be serialized to the journal somehow...  Anyway, we can deal with that in a separate PR, but maybe add a TODO here?

Reply:
Sounds good; will add a TODO. I thought about how these would work a little bit, but decided that would be best to stay in a separate PR, and put in `nil` everywhere.